### ISSUE TITLE
Books not showing in Series

### ISSUE THREAD

**Issue by Dillsburg** [eleybourn/Book-Catalogue#904]
When scanning books that are in a series, they are no longer marked as being in a series. They are not numbered and not listed as being in any series. This can be done manually, but is time consuming.

This change just started today.

**Comment by toniaaymond** [comment]
Having the same issue. It's extremely time consuming to manually enter it for each book. This wasn't an issue before the update

**Comment by Dillsburg** [comment]
I did some more trouble-shooting with no resolution. The best we can do is manually set the series. We don't always easily know the series number.

One thing I did try before that was going into the "Automatically Update Fields" option under Administration Functions. I un-selected everything EXCEPT "Series". It didn't take too long and I was hopeful, but did *not* appear to work.

Very thankful for this great App (the best I have found!) - just wishing the Series field would work again for us.

Phone: Galaxy S10e
Android Version: 12
Book Catalogue v. 6.0.7-distplay

**Comment by ichimatsuri** [comment]
I also have the issue where some book series automatically  get put in as "novel" or other similar things

**Comment by Dparker982** [comment]
Has anyone found a way to solve this?
The series option was THE reason I use BC

**Comment by toniaaymond** [comment]
Nope. I moved into a different app. Handy Library, but it's only on Android
devices at the moment

**Comment by eleybourn** [comment]
I know this is an old issue, but changes to the GoodReads and Amazon APIs has limited the ability to search their databases. I've built new integrations into the latest version that improves the ability to search series.
